mycity/mycity/mycity_controller.py

```diff
@@ -173,6 +174,37 @@ def on_session_ended(mycity_request):
     # add cleanup logic here
 
 
+def get_help_response(mycity_request):
+    """
+    Informs the user of currently functionality. This is triggered on
+    AMAZON.HelpIntent.
+
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseDataModel object that will initiate
+        a help process on the user's device
+    """
+    print(
+        LOG_CLASS,
+        '[method: get_help_response]'
+    )
+    mycity_response = MyCityResponseDataModel()
+    mycity_response.session_attributes = mycity_request.session_attributes
+    mycity_response.card_title = "Help"
+    mycity_response.output_speech = \
+        "You can request information on trash and recycling pickup by saying, " \
+        "When is my trash day at 'say your address here'. This is currently " \
+        "only available for Boston residents. We're working on adding more cities. " \
```

Comment:
I don't think we need to mention other cities since Boston is in the app name. We can add an intro that is something like "I can provide information about Boston city services".